README.md

```diff
@@ -1 +1,94 @@
-# playwrightTutorial
\ No newline at end of file
+# playwrightTutorial
```

Comment:
Content of the file is great, I would suggest playing with formatting a bit more: https://www.markdownguide.org/basic-syntax/#code for improved readability.